demo/index.js

```diff
@@ -256,14 +329,15 @@ import { OpenSheetMusicDisplay } from '../src/OpenSheetMusicDisplay/OpenSheetMus
         if (!isCustom) {
             str = sampleFolder + selectSample.value;
         }
-        zoom = 1.0;
+        // zoom = 1.0;
```

Comment:
awesome. not resetting the zoom level between sample changes is more correct anyways. if anything, we need a reset zoom button in the (main) demo.